model/userrating.js

```diff
@@ -0,0 +1,15 @@
+function UserRating(userid,userName,rating) {
```

Comment:
jsdocs. Please follow the jsdocs standards we use for all projects. Each function must have a full jsdoc docstring with `@param` for each parameter and `@returns` if it returns. Also, this module should be called `user-rating.js` and it should have a docstring at the top of it explaining what it is for.